.mergify.yml

```diff
@@ -0,0 +1,24 @@
+pull_request_rules:
+  # automerge backports if CI successfully ran
+  - name: automerge backport release-1.1
+    conditions:
+      - author=mergify[bot]
+      - base=release-1.1
+      - label!=no-not-merge
+      - 'status-success=DCO'
+      - 'status-success=continuous-integration/jenkins/pr-head'
+    actions:
+      merge:
+        method: rebase
+        rebase_fallback: merge
+        strict: smart
+      dismiss_reviews: {}
+      delete_head_branch: {}
+# Trigger backport PRs based on label
+  - actions:
+      backport:
+        branches:
+        - release-1.1
+    conditions:
+      - label=backport-release-1.1
+    name: backport release-1.1
```

Comment:
Do we need to create a new rule every time we create a new release branch? It would be nice if we could look at the label and extract the branch name.

Reply:
Yes, one rule for one branch AFAIK.

Reply:
@leseb Could you add a note about this requirement for new releases to the development flow doc? Not that we forget :slightly_smiling_face: